static/app/data/platformCategories.tsx

```diff
@@ -499,7 +499,17 @@ const customMetricBackendPlatforms: readonly PlatformKey[] = [
 ];
 
 const customMetricFrontendPlatforms: readonly PlatformKey[] = [
+  'android',
   'electron',
+  'java',
+  'java-android',
+  'java-appengine',
+  'java-log4j',
+  'java-log4j2',
+  'java-logback',
+  'java-logging',
+  'java-spring',
+  'java-spring-boot',
```

Comment:
Looks good but i think these should go to `customMetricBackendPlatforms` instead